### PR TITLE
SDK install flow fixes and bit flip (#764).

### PR DESCRIPTION
**Pull request by pq**
* implement `hasGit` and flip the bit
* fix local filesystem refresh issues w/ newly cloned SDKs

See: #764.

@devoncarew

### CODE REVIEW

File: src/io/flutter/module/FlutterModuleBuilder.java [hunk 1]
```diff
@@ -269,7 +269,19 @@ public void dispose() {
...
     @Nullable
     public FlutterSdk getFlutterSdk() {
-      return FlutterSdk.forPath(myPeer.getSdkComboPath());
+      final String sdkPath = myPeer.getSdkComboPath();
+
+      //Ensure the local filesystem has caught up to external processes (e.g., git clone).
+      if (!sdkPath.isEmpty()) {
+        try {
+          LocalFileSystem
+            .getInstance().refreshAndFindFileByPath(sdkPath);
```
Comment (devoncarew) [inline src/io/flutter/module/FlutterModuleBuilder.java]: Should this refresh instead be done at the end of the call to `flutter precache`?

File: src/io/flutter/actions/InstallSdkAction.java [hunk 2]
```diff
@@ -30,6 +31,9 @@
 @SuppressWarnings("ComponentNotRegistered")
 public class InstallSdkAction extends DumbAwareAction {
 
+  //TODO(pq): add support for "git.exe"
+  private static final String GIT_EXECUTABLE = "git";
```
Comment (devoncarew) [inline src/io/flutter/actions/InstallSdkAction.java]: This may do it? `SystemInfo.isWindows ? "git.exe" : "git"`

**Comment by devoncarew** [comment]
I suspect that we'll want to do that file system refresh at the end of the sdk step (after the sdk's written out) instead of on every access to `getFlutterSdk()`.

lgtm

File: src/io/flutter/module/FlutterModuleBuilder.java (hunk 1 above)
Comment (pq) [inline src/io/flutter/module/FlutterModuleBuilder.java]: I tried this a few different ways but couldn't get the threading right.  The good news is that this method is only accessed in this wizard step so it's effectively the same as calling it after `flutter precache`.

File: src/io/flutter/actions/InstallSdkAction.java (hunk 2 above)
Comment (pq) [inline src/io/flutter/actions/InstallSdkAction.java]: 👍 

I'm slightly hesitant to flip this on for Windows w/o doing some testing first.  Happy to defer to you if you say bombs away though.

File: src/io/flutter/module/FlutterModuleBuilder.java (hunk 1 above)
Comment (devoncarew) [inline src/io/flutter/module/FlutterModuleBuilder.java]: Ah! I didn't check which class this was in. lgtm (though, perhaps reduce the visibility of the method?)

Comment (pq) [inline src/io/flutter/module/FlutterModuleBuilder.java]: Good call.  Done!

File: src/io/flutter/actions/InstallSdkAction.java (hunk 2 above)
Comment (pq) [inline src/io/flutter/actions/InstallSdkAction.java]: FYI: I didn't flip this (in the absence of an easy way to test today) but can in a follow-up if you'd like to see it.

**Pull request merged**